src/Entity/RulesComponent.php

```diff
@@ -142,6 +154,20 @@ public function label() {
   }
 
   /**
+   * Returns the description.
+   */
+  public function getDescription() {
+    return $this->description;
+  }
+
+  /**
+   * Returns the tag.
+   */
+  public function getTag() {
```

Comment:
Don't know howto do this.
Klausi commented on protected $tag (in class RulesComponent line ~70)

```
  /**
   * The "tags" of a Rules component.
   *
   * The tags are stored as a single string, though it is used as multiple tags
   * for example in the rules overview.
   *
   * @var string
   */
  protected $tag = '';
```


Prefer to move to follow up ticket.